src/components/container/sticky-bar/sticky-bar.scss

```
@@ -0,0 +1,38 @@
+@use '../../core/styles' as sbb;
+
+:host {
+  @include sbb.box-sizing;
+
+  --sbb-sticky-bar-padding-block: var(--sbb-spacing-responsive-xxxs);
+  --sbb-sticky-bar-animation-duration: var(--sbb-animation-duration-5x);
+
+  display: contents;
+}
+
+.sbb-sticky-bar__wrapper {
+  box-shadow: none;
+  position: sticky;
+  inset-block-end: 0;
+  display: inline-block;
```

It should not really change something, but this is clearly a block element for me :)
```suggestion
  display: block;
```